### DISCUSSION TITLE
Hide/Show the mutliple .glb files

### DISCUSSION THREAD

**Discussion by Arumugamsix** [google/model-viewer#4601] [Q&A]
How to hide and show the multiple .glb files. 

my idea is to develop a car configurator like changing the alloy wheels, changing the seats and tires. changing the car variant not complete car some parts in the exterior of the vehicle.

kindly suggest or share the methods to do

**Comment by elalish** [comment]
We have an example you can follow: https://modelviewer.dev/examples/scenegraph/#hideShowMeshVariantsExample - just make each separate part have its own material - and ideally give them unique names so they are easy to identify.
